nanoDFT/nanoDFT.py

```diff
@@ -396,6 +397,7 @@ def nanoDFT_options(
     mol_str = args["mol_str"]
     del args["mol_str"]
     args = Namespace(**args)
+    args = namedtuple('immutable',vars(args).keys())(**vars(args)) # make immutable
```

Comment:
Can we have a bit descriptive typename. Like `DFTOptionsImmutable`? I believe that would help if there is an error raise somewhere in the code using it

Reply:
Thank you, this looks much better now!